package.json

```diff
@@ -78,7 +80,9 @@
     "react-geocode": "^0.2.0",
     "react-is": "^16.12.0",
     "react-native": "https://github.com/expo/react-native/archive/sdk-37.0.0.tar.gz",
+    "react-native-dotenv": "^0.2.0",
```

Comment:
@kennethlien @wangannie This as well - are we using it? I don't feel like we should be.. `environment.js` should handle any env configs needed. Better to have single source of truth